schemaless/create_uuid_map.py

```diff
@@ -67,9 +68,13 @@ def from_files(cls, schemaless_file, uuid_map_file):
                                 children.append(child_fk)
 
                 if source == PTS.NAME:
-                    if record['permit_number'] in permit_number_to_ppts_fk:
-                        parents.extend(permit_number_to_ppts_fk[
-                            record['permit_number']])
+                    permit_number = record['permit_number']
+                    if permit_number in permit_number_to_ppts_fk:
+                        parents.extend(permit_number_to_ppts_fk[permit_number])
+                    elif permit_number in permit_number_to_pts_fk:
+                        if permit_number_to_pts_fk[permit_number][0] == fk:
```

Comment:
what's going on here?

Reply:
Wanted to make sure that PTS records with the same permit number (but no associated PPTS records) have the same UUID. So I just thought I'd pick the first record with a given permit number and make it the parent of any other PTS records with the same permit number. 

I thought this logic could be weird and unnecessary in terms of graph structure if there's an associated parent PPTS record though, in which case I just make all PTS records direct children of the PPTS record. Thoughts?

Reply:
Ah, I see. Ok, that sounds reasonable. Can you add a comment explaining it?

Reply:
Done